src/networking.c

```diff
@@ -405,6 +405,28 @@ void addReplyErrorLength(client *c, const char *s, size_t len) {
 
 /* Do some actions after an error reply was sent (Log if needed, updates stats, etc.) */
 void afterErrorReply(client *c, const char *s, size_t len) {
+    /* Update per command error stats.
+     * If the command was rejected no need to flag it
+     * as a failed count also. */
+    if (c->cmd && !(c->cmd->flags & CMD_ERR_REJECTED)) {
```

Comment:
where does CMD_ERR_REJECTED get reset?

Reply:
you're totally right @madolson. I've fixed it and added a test to confirm it's working as expected